src/ticketing/utils.ts

```diff
@@ -47,8 +47,11 @@ export function hasValidRightNowTravelRight(
   );
 }
 
-function isValidRightNowTravelRight(travelRight: TravelRight, now: number) {
-  // Unknown travel rights are not active
+export function isValidRightNowTravelRight(
```

Comment:
I guess we don't need the "RightNow" parts of names anymore, if "valid" is always relative to `now`

Reply:
This wasn't ment to be included for now, so I removed it. But you are right, we don't need to "RightNow" part, and hopefully we don't need this function at all after we start using `getAvailabilityStatus` around the app.